### PR TITLE
Data comparisons

### PR DESCRIPTION
**Pull request by JacksonLChin**
Plots predictions for training and validation samples using various data sources

### CODE REVIEW

File: tfac/figures/figure5.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+from tfac.figures.figureCommon import getSetup
+from tfac.dataImport import form_tensor, import_cytokines, import_rna
+from tfac.predict import predict_known, predict_unknown
+from tfac.tensor import perform_CMTF
+
+OPTIMAL_SCALING = 32
```
Comment (aarmey) [inline tfac/figures/figure5.py:11]: This should be defined in one place and then imported in the various figures.

File: tfac/figures/figure5.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+    Returns:
+        data_types (list[tuple]): data sources and their names
+        patient_data (pandas.DataFrame): patient metadata
+    """
+    plasma_cyto, serum_cyto = import_cytokines()
+    tensor, matrix, patient_data = form_tensor(OPTIMAL_SCALING)
+    patient_data = patient_data.loc[:, ['status', 'type']]
+
+    components = perform_CMTF(tensor, matrix, 9)
```
Comment (aarmey) [inline tfac/figures/figure5.py:38]: Leave off component number so that it uses the default, so this is defined in one place.

**Review by aarmey — CHANGES_REQUESTED** [review]
Two small changes then this should be ready.

**Comment by JacksonLChin** [comment]
Just pushed a new commit that should address the requested changes!

**Review by aarmey — APPROVED** [review]

**Pull request merged**